database/word_frequency_table.sql

```diff
@@ -0,0 +1,8 @@
+USE flexicon_words;
+CREATE TABLE word_frequency(
+   word_id INT NOT NULL AUTO_INCREMENT,
```

Comment:
We talked a little about this, but I'm not sure if we settled on a table design for this. Maybe WordFrequency has word, usage_frequency, frequency_percentile, and frequency_rank? Do we need a submission_date? Technically this table should be kind of like a picklist, so the date when the row is created is kind of irrelevant.